zebra-consensus/src/transaction.rs

```diff
@@ -136,8 +136,7 @@ where
                 | Transaction::V2 { .. }
                 | Transaction::V3 { .. }
                 | Transaction::V5 { .. } => {
-                    tracing::debug!(?tx, "got transaction with wrong version");
-                    Err(TransactionError::WrongVersion)
+                    unimplemented!("v5 transaction validation as specified in ZIP-216, ZIP-224, ZIP-225, and ZIP-244")
                 }
```

Comment:
Please split the V5 match from the other matches.

V1-V3 are not `unimplemented!()`, they're invalid.

Reply:
Our goal here is:
* keep the same behaviour for `V1`, `V2`, and `V3`
* mark all `V5` with `unimplemented!()` (or implement them)

Reply:
Sorry about that, i fixed it https://github.com/ZcashFoundation/zebra/pull/1824/commits/8ac970c5f1ff9c524ef0e0f481d249ff531a1071